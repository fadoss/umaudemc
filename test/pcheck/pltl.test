# Probabilities of temporal formulae (LTL formulae, for PRISM)
#+ req: prism

umaudemc pcheck coin head '[] <> tail'
umaudemc pcheck coin head '[] <> tail' --assign uniform
umaudemc pcheck coin head '[] <> tail' --assign metadata
umaudemc pcheck coin head '[] <> tail' --assign 'uaction(ttail=3, thead=7)'
umaudemc pcheck coin head '[] <> tail' --assign @ite.assign
umaudemc pcheck coin head '[] <> tail' --assign "term(inertia(L, R))"
umaudemc pcheck coin head '[] <> tail' 'choice(3 : ttail, 7 : thead)' --assign strategy
umaudemc pcheck coin head '[] <> tail' 'choice(3 : ttail, 7 : thead) *' --assign strategy
umaudemc pcheck coin head '[] <> tail' 'choice(3 : ttail, 7 : thead) ; choice(2 : ttail, 3 : thead)' --assign strategy
