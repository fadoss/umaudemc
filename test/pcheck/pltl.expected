Result: 1.0
Result: 1.0
Result: 1.0
Result: 1.0
Result: 1.0
Result: 1.0
Result: 0.3
Result: 0.0 to 1.0
Result: 0.39999999999999997
